src/utils/search.js

```diff
@@ -22,7 +22,7 @@ export function createPreprintQs(
 
   if (text != null && text != '') {
     ui.set('q', text);
-  } else if (text === null) {
+  } else if (text === text) {
```

Comment:
Seems like this could just be an `else` instead of `else if`, since wouldn't this always evaluate to true?